samples/ActionConstraintSample.Web/project.json

```diff
@@ -12,9 +12,12 @@
     "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*"
   },
   "frameworks": {
-    "net451": { },
-    "dnxcore50": {
-      "imports": "portable-net451+win8",
+    "net451": {},
+    "netstandardapp1.5": {
```

Comment:
It's because soon, `netstandardapp` will actually refer to a specific set of APIs that will be assumed to be available in the shared framework. `netstandard` refers to the standard library alone.